### DISCUSSION TITLE
Welcome to the OpenVic Discussions!

### DISCUSSION THREAD

**Discussion by Spartan322** [OpenVicProject/OpenVic#173] [Announcements]
In order to better document and organize user contributions, I'm setting up discussions so as to try and move user submissions more over to a place where it can be better documented. Keeping everything to the Discord is very poor when it comes to documentation and indexing. It is much easier to get clearer ideas for each contribution when its documented via Github, as well it better associated with development of OpenVic compared to it being just a part of the community. It can also be referenced in commits and issues whereas this cannot be done with Discord. For now this will remain less considered until formally announced on the Discord.

**Comment by valahdev** [comment]
Thanks for setting this up, Spartan!

A reminder for everyone that you are not expected to be active here, it is your choice which medium (forum) you would like to use to share ideas, point faults, etc. As Spartan mentioned above, this place will foster easier communication that will allow you to point directly to sections of the code from our repository, but ultimately, we are happy to see activity on whichever place you choose.

Motionsense
